README.md

```diff
@@ -30,6 +30,9 @@ Options:
     --web-security <bool>  Enables PhantomJS web security and forbids
                            cross-domain XHR (default is true)
     --brout                Assume brout is part of the JS
+    --ignore-ssl-errors <bool>  Tell PhantomJS web security and forbids
+                                cross-domain XHR (default is false)
```

Comment:
This doesn't seem like the right description. Is this a copy-paste from the `--web-security` option?